<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_dragKeep">
          <property name="text">
           <string>KeepCurrent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dragSin">
          <property name="text">
           <string>Sin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dragPoints">
          <property name="text">
           <string>Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_dragSym">
          <property name="text">
           <string>Symetric</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="dragDeadband">
        <property name="decimals">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_pitchSym">
          <property name="text">
           <string>Symetric</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_pitchPoints">
          <property name="text">
           <string>Points</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="dragMax">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>1.280000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QTextEdit" name="textEditDragPoints"/>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="dragMin">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.004000000000000</double>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>PitchPoints</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>LiftPoints</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Multiplier</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_liftSym">
          <property name="text">
           <string>Symetric</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_liftPoints">
          <property name="text">
           <string>Points</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="15" column="1">
       <widget class="QTextEdit" name="textEditPitchPoints"/>
      </item>
      <item row="16" column="1">
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update Values</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QPushButton" name="updateGraphButton">
        <property name="text">
         <string>Update Graph only</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Drag</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="liftMulti">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Deadband</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="textEditLiftPoints"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>DragPoints</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="17" colspan="2">
       <layout class="QGridLayout" name="gridLayout_3"/>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lift</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Max alphas, x-plane will start adding random tubulence at these angles </string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="18" column="2">
       <widget class="QDoubleSpinBox" name="dh_aMin">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="18" column="3">
       <widget class="QDoubleSpinBox" name="dh_aMax">
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionImport_X_Plane_Airfoil"/>
    <addaction name="actionExport_X_Plane_Airfoil"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionImport_X_Plane_Airfoil">
   <property name="text">
    <string>Import X-Plane Airfoil</string>
   </property>
  </action>
  <action name="actionExport_X_Plane_Airfoil">
   <property name="text">
    <string>Export X-Plane Airfoil</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
